controllers/runner_controller.go

```diff
@@ -530,6 +530,10 @@ func (r *RunnerReconciler) newPod(runner v1alpha1.Runner) (corev1.Pod, error) {
 		},
 	}
 
+	if dockerdInRunner {
+		pod.Spec.Containers[0].Command = []string{"startup.sh", fmt.Sprintf("--mtu %d", *runner.Spec.DockerMTU)}
```

Comment:
It looks like to fail when DockerMTU is nil. Probably we'd better enclose this inside a if block?

```suggestion
		if mtu := runner.Spec.DockerMTU; mtu != nil {
			pod.Spec.Containers[0].Command = []string{"startup.sh", fmt.Sprintf("--mtu %d", *runner.Spec.DockerMTU)}
		}
```